### ISSUE TITLE
Feature request: Swedish NER

### ISSUE THREAD

**Issue by EmilStenstrom** [explosion/spaCy#9969]
I didn't find a way to post feature requests, I hope it's OK to post it here? 

I was hoping that spacy would be willing to add support for NER in Swedish.

For this, you need a Swedish dataset in IOB2 format. And I have a way to get a file like that using the SUC 3.0 corpus, and a small converter script that I wrote: https://github.com/EmilStenstrom/suc_to_iob

The license for the corpus us CC BY-SA, and it does not affect the license of the released model files (see the repo above for licensing clarifications).

How can I help to get this done? :)

**Comment by polm** [comment]
Thanks for the suggestion! For adding new languages, we generally want to be able to add a full pipeline like for English or other languages, which means:

- word vectors
- POS tagger
- Universal Dependencies parser
- NER

We have not released an official model that's just NER before. It looks like SUC also has POS info. It seems to have some kind of dependency parse data but I think it's not UD?

Another thing is that for official models, we must have access to the underlying corpus - we need to be able to train the models ourselves as we update spaCy, for example. I see that SUC is freely available if you sign an agreement, but can anyone (such as ourselves) sign that and get access?

There are other considerations, but those are some of the broad requirements for adding a new official model. 

Another option is that you can train and release a model yourself - since models are just Python packages, you can distribute your model via Github releases or Huggingface Hub or other places, just like we do for the core models. If you're happy with a pure NER model this might be a good approach.

If all that sounds good and you'd like to work on making an official model we could certainly discuss it. Maybe as a first step you could train a spaCy NER model on the corpus and see what kind of scores you get.

**Comment by EmilStenstrom** [comment]
Thanks for the quick reply! 

Isn't dependency parse data what is available in the UD datasets for swedish? SUC is something else, and unaffiliated with UD. Maybe the Swedish UD data could be used to train a POS and dependency parser, and SUC for NER?

The SUC dataset is freely available with sentences shuffled, and a permissible license. I wrote a little bit about that here: https://github.com/EmilStenstrom/suc_to_iob/#clarification-on-the-suc-30-license

When looking around I found https://github.com/Kungbib/swedish-spacy which already have the trained models (all four parts above!) that you references. It was just not included in spacy by default, so a bit hard to find. Hopefully in the future this can be included as in official spacy model. I'll close this issue and hope for that instead.
